lbrynet/daemon/Publisher.py

```diff
@@ -54,6 +55,42 @@ def create_and_publish_stream(self, name, bid, claim_dict, file_path, claim_addr
     @defer.inlineCallbacks
     def publish_stream(self, name, bid, claim_dict, claim_address=None, change_address=None):
         """Make a claim without creating a lbry file"""
+        log.info('Starting publish for %s', name)
+
+        source = claim_dict['stream']['source'].get('source', None)
+        source_type = claim_dict['stream']['source'].get('sourceType', None)
```

Comment:
I think you should also rename the `ClaimDict.source_hash` property to `ClaimDict.source_value`, since with these changes it could be a url and not a BTIH or a SD hash